sdk/deviceupdate/Azure.Iot.DeviceUpdate/README.md

```diff
@@ -0,0 +1,75 @@
+
+  [Source code](https://github.com/Azure/azure-sdk-for-sdk/tree/master/sdk/deviceupdate/Azure.Iot.DeviceUpdate/src/) | [Package](https://www.nuget.org/packages/Azure.DeviceUpdate) | [Product documentation](https://github.com/Azure/adu-private-preview/blob/master/docs/adu-overview.md)
+
+
+## Getting started
+
+The complete Microsoft Azure SDK can be downloaded from the [Microsoft Azure Downloads](https://azure.microsoft.com/downloads/?sdk=net) page and ships with support for building deployment packages, integrating with tooling, rich command line tooling, and more.
+
+For the best development experience, developers should use the official Microsoft NuGet packages for libraries. NuGet packages are regularly updated with new functionality and hotfixes.
+
+### Prerequisites
+
+- Microsoft Azure Subscription: To call Microsoft Azure services, you need to create an [Azure subscription](https://azure.microsoft.com/free/)
+- Device Update for IoT Hub instance
+- Azure IoT Hub instance
+
+### Install the package
+
+Install the Device Update for IoT Hub client library for .NET with [NuGet](https://www.nuget.org/ ):
+
+```PowerShell
+dotnet install Azure.Iot.DeviceUpdate --version 1.0.0-beta.1
+```
+
+### Authenticate the Client
+
+In order to interact with the Device Update for IoT Hub service, you will need to create an instance of a [TokenCredential class](https://docs.microsoft.com/dotnet/api/azure.core.tokencredential?view=azure-dotnet) and pass it to the constructor of your UpdateClient, DeviceClient and DeploymentClient class.
+
+## Key concepts
+
+Device Update for IoT Hub is a managed service that enables you to deploy over-the-air updates for your IoT devices. The client library has three main components:
+- **UpdatesClient**: update management (import, enumerate, delete, etc.)
+- **DevicesClient**: device management (enumerate devices and retrieve device properties)
+- **DeploymentsClient**: deployment management (start and monitor update deployments to a set of devices)
+
+You can learn more about Device Update for IoT Hub by visiting [Device Update for IoT Hub](https://github.com/Azure/adu-private-preview/tree/release/v0.2.0-private-preview).
+
+## Examples
+
+You can familiarize yourself with different APIs using [Samples](https://github.com/Azure/azure-sdk-for-sdk/tree/master/sdk/deviceupdate/Azure.Iot.DeviceUpdate/samples).
+
+## Troubleshooting
+
+All Device Update for IoT Hub service operations will throw a RequestFailedException on failure with helpful ErrorCodes.
+
+For example, if you use the `GetUpdateAsync` operation and the model you are looking for doesn't exist, you can catch that specific [HttpStatusCode](https://docs.microsoft.com/dotnet/api/system.net.httpstatuscode?view=netcore-3.1) to decide the operation that follows in that case.
+
+```csharp
```

Comment:
Please use snippets https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#updating-sample-snippets